CMakeLists.txt

```diff
@@ -706,6 +704,7 @@ function (treat_warnings_as_errors dirs)
   endforeach()
 endfunction()
 
+include(version.cmake)
```

Comment:
For the sake of minimizing diffs, should just put this include back up on line 690, where it was in the original file.

Reply:
ok